src/components/Text/Text.tsx

```diff
@@ -36,6 +36,7 @@ const Text = ({ size, color, children, weight, align }: TextProps) => {
   }
 
   switch (color) {
+    case undefined:
```

Comment:
Why case undefined as empty? Shouldn't it just use the default/white or something?

Reply:
Yeah it will fall through to the White case

Reply:
Its essentially doing `case undefined || "primary"` since you cant do an or statement in a switch case

Reply:
ohh neat